.circleci/config.yml

```
@@ -2,18 +2,20 @@ version: 2
 jobs:
   MacOS:
     macos:
-      xcode: "9.0"
+      xcode: "9.3.0"
     steps:
       - checkout
       - restore_cache:
           keys:
-            - v1-spm-deps-{{ checksum "Package.swift" }}
+            - v3-spm-deps-{{ checksum "Package.swift" }}
```

```suggestion
            - v1-spm-deps-{{ checksum "Package.swift" }}
```

Updating `Package.swift` to 4.2 should allow us to go back to v1 and v2.